engine/deployments/helm/tiflow/templates/cases.yaml

```diff
@@ -0,0 +1,36 @@
+{{- if .Values.cases.enabled -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Release.Name }}-chaos-test-case
+  namespace: {{ .Release.Namespace }}
+spec:
+  template:
+    spec:
+{{- if .Values.cases.imagePullSecrets }}
+      imagePullSecrets:
+{{ toYaml .Values.cases.imagePullSecrets | indent 6 }}
+{{- end }}
+      initContainers:
+        - name: wait-tiflow-cluster
+          image: busybox
+          command: ['sh', '-c', 'sleep 60']
```

Comment:
Will we use a health check API later?

Reply:
Yes, we will add API check later.

Reply:
@gozssky prepares to add readiness API this week, can have a discussion about it.